draft/2022-05-18-this-week-in-rust.md

```diff
@@ -72,15 +72,43 @@ If you are a Rust project owner and are looking for contributors, please submit
 
 <!-- Perf results go here -->
 
-### Call for Testing
+### Call for Testing (*New!*)
 
 An important step for RFC implementation is for people to experiment with the
 implementation and give feedback, especially before stabilization.  The following
-RFCs are at point where user testing is needed before moving forward:
+RFCs would benefit from user testing before moving forward:
 
-<!-- Pre-Stabilization RFCs go here -->
+* [RFC: Deduplicate Cargo workspace information](https://github.com/rust-lang/rfcs/pull/2906)
+    - [Tracking Issue](https://github.com/rust-lang/cargo/issues/8415)
+    - [Testing steps](https://github.com/rust-lang/cargo/blob/master/src/doc/src/reference/unstable.md#testing-notes)
```

Comment:
What're your thoughts on this structure? I know this is what I had but I was worried it was too different of a format from other things around it. 

As well should we use tracking issues or summary posts (where there are ones). Summary posts may give a more in-depth look than just the tracking issue i.e. [Tracking Issue](https://github.com/rust-lang/cargo/issues/8415) vs [Sumamry](https://github.com/rust-lang/cargo/issues/8415#issuecomment-1112618913)

Reply:
That's perfect! I am updating some of the cargo docs to include the updates to TWiR. This comment as well as a few others will be helpful as I write those